prereise/gather/hiflddata/tests/test_plant_agg.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from unittest.mock import Mock
+
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+from prereise.gather.hiflddata.plant_agg import (
+    Cal_P,
+    Clean_p,
+    Loc_of_plant,
+    LocOfsub,
+    Plant_agg,
+)
+
+
+def test_clean_p():
+    csv_data_path = "General_Units.csv"
+    csv_data = {"STATE": ["AL", "AR"], "STATUS": ["OP", "Other"]}
+    pd.read_csv = Mock(return_value=pd.DataFrame(data=csv_data))
```

Comment:
Over-writing the pandas `read_csv` function is what's causing other tests to break, this needs to be removed. You can refactor the cleaning functions to take a dataframe, rather than a path to a CSV file.

Reply:
I have applied `@patch("prereise.gather.hiflddata.plant_agg.pd.read_csv")` to the unit test. it will be only effective within the module.